src/components/RefreshPuzzleButton/refreshPuzzleButton.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
```

Comment:
I'm a little bit dubious about what we have gained by separating the refresh button into its own component.

A common pattern is to create a generic `Button.jsx` component that has base button styling, and into which you can pass a click handler and text to display. But as it stands now, it doesn't improve the structure of our app at all to have it separate. What are your thoughts? Did you have a specific goal in mind in separating this button?